### PR TITLE
Starting point for python boilerplate

### PR DESCRIPTION
**Pull request by AlexMoffat**
My attempt at an equivalent to our java and node examples. I'd like reviews and for some people to try this out by checking out the branch and following the readme instructions.

### CODE REVIEW

**Comment by BlakeGilstrap** [comment]
Had no issues w/ instructions:

```
flask run                                                                                                                                                                                                                                                                  ─╯
 * Serving Flask app 'wordcount' (lazy loading)
 * Environment: development
 * Debug mode: on
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 104-741-888
127.0.0.1 - - [23/May/2021 16:52:21] "GET /helloworld HTTP/1.1" 200 -
```

File: python-flask/wordcount/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    if test_config is None:
+        app.config.from_object('config')
+    else:
+        app.config.from_mapping(test_config)
+
+    @app.route("/helloworld")
+    def hello_world():
+        return jsonify({"text": app.config['HELLO_WORLD']})
+
+    return app
```
Comment (stv8) [inline python-flask/wordcount/__init__.py:20]: I think we should separate the app creation and route definitions, something like

```python
app = create_app()

@app.route("/helloworld")    
def hello_world():        
    return jsonify({"text": app.config['HELLO_WORLD']})
```

**Review by stv8 — APPROVED** [review]
One small comment, otherwise seems fine

**Review by stv8 — APPROVED** [review]
Instructions worked great with new setup

**Review by BlakeGilstrap — APPROVED** [review]
LGTM. Could perhaps add `-> Any` for return type hints but perhaps we save that for a more fleshed out take home version.

**Pull request merged**